packages/vscode-extension/src/handlers.ts

```diff
@@ -1419,10 +1419,16 @@ export async function backendExtensionsInstallHandler(): Promise<string | undefi
  */
 export async function getFuncPathHandler(): Promise<string> {
   try {
+    if (!vscodeHelper.isFuncCoreToolsEnabled()) {
+      return "";
```

Comment:
Will `path.delimiter` be more compatible?

Reply:
Added